psl-core/src/main/java/org/linqs/psl/util/RandUtils.java

```diff
@@ -25,6 +26,9 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Random;
+import org.apache.commons.math3.distribution.GammaDistribution;
```

Comment:
Can't we just implement a Gamma distribution instead of importing a fairly large library?
(Pending later comments that remove other uses of this library.)

Reply:
I added nextGamma() to RandUtils which implements the algorithm by Marsaglia and Tsang (2000a): https://dl.acm.org/doi/10.1145/358407.358414 . Specifically implemented the version of the algorithm in: http://statweb.stanford.edu/~owen/mc/Ch-nonunifrng.pdf